src/app/pages/common/ix-forms/services/ix-formatter.service.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { createServiceFactory, SpectatorService } from '@ngneat/spectator';
+import { IxFormatterService } from 'app/pages/common/ix-forms/services/ix-formatter.service';
+
+describe('IxFormatterService', () => {
+  let spectator: SpectatorService<IxFormatterService>;
+
+  const createService = createServiceFactory({
+    service: IxFormatterService,
+  });
+
+  beforeEach(() => {
+    spectator = createService();
+  });
+
+  it('should instantiate', () => {
+    expect(spectator).toBeTruthy();
+  });
+
+  describe('memorySizeFormatting', () => {
+    it('should return empty string when invalid size string is passed', () => {
+      const formatted = spectator.service.memorySizeFormatting('2u');
+      expect(formatted).toBe('');
+    });
+
+    it('should return normalized string when valid size string is passed', () => {
```

Comment:
It shouldn't. It's a job for parser.

Reply:
By 'normalized' I mean with proper spacing and capitalization. It is only formatting `2gb` to `2 GiB`.

Reply:
Why would we expect '2gb' in this function? Haven't we agreed that this function will only deal with proper numeric values and it would be parse function that would deal with loose strings?